assets/js/base/components/tabs/index.js

```diff
@@ -20,12 +20,10 @@ const Tabs = ( {
 	instanceId,
 	id,
 } ) => {
-	const initialTab =
-		( initialTabName || !! tabs.length ) &&
-		`${ instanceId }-${ initialTabName ? initialTabName : tabs[ 0 ].name }`;
-	const tabState = useTabState( {
-		selectedId: initialTab,
-	} );
+	const initialTab = initialTabName
+		? { selectedId: `${ instanceId }-${ initialTabName }` }
+		: undefined;
+	const tabState = useTabState( initialTab );
```

Comment:
weird, I assumed this will always default to the preselected value or the first tab, reakit handles this logic internally `tabs[ 0 ].name`.